common/src/main/java/com/wynntils/models/gear/GearModel.java

```diff
@@ -48,9 +51,10 @@ public final class GearModel extends Model {
 
     private final GearChatEncoding gearChatEncoding = new GearChatEncoding();
     private final Map<GearBoxItem, List<GearInfo>> possibilitiesCache = new HashMap<>();
+    private final Map<String, List<SetInstance>> activeSetsCache = new HashMap<>();
 
-    public GearModel() {
-        super(List.of());
+    public GearModel(SetModel setModel) {
```

Comment:
This dependency is not needed, since you don't use the set model from the constructor. I think you felt that this model depends on the set data, but this is not the way to ensure the data is loaded, since this just ensures that `SetModel`'s constructor ran, which loads the data **asynchronously**.